server/server.go

```diff
@@ -292,13 +294,18 @@ func (s *server) Start(ctx context.Context, req *rpcpb.StartRequest) (*rpcpb.Sta
 		zap.Int32("pid", pid),
 		zap.String("rootDataDir", rootDataDir),
 		zap.String("pluginDir", pluginDir),
-		zap.String("nodeConfig", nodeConfigParam),
+		zap.String("defaultNodeConfig", defaultNodeConfig),
 	)
 
 	if s.network != nil {
 		return nil, ErrAlreadyBootstrapped
 	}
 
+	if len(customNodeConfigs) > 0 {
```

Comment:
I don't understand why this is needed. Also shall avoid it (not force the user to given custom configs for all nodes)